src/qt/managenamespage.cpp

```diff
@@ -120,6 +137,42 @@ void ManageNamesPage::onCopyValueAction()
     GUIUtil::copyEntryData(ui->tableView, NameTableModel::Value);
 }
 
+void ManageNamesPage::onConfigureNameAction()
+{
+    QModelIndexList indexes = GUIUtil::getEntryData(ui->tableView, NameTableModel::Name);
+
+    if (indexes.isEmpty())
+        return;
+
+    if (indexes.size() != 1)
+        return;
+
+    WalletModel::UnlockContext ctx(walletModel->requestUnlock ());
+    if (!ctx.isValid ())
+        return;
+
+    const QModelIndex &index = indexes.at(0);
+    const QString &name = index.data(Qt::EditRole).toString();
+    const std::string &strName = name.toStdString();
+    const QString &initValue = index.sibling(index.row(), NameTableModel::Value).data(Qt::EditRole).toString();
```

Comment:
I'm not too familiar with Qt, but I think in these cases, the returned value is actually a new instance and not a reference?  I think the C++ code may still work (and just accommodate for that automatically), but it seems cleaner to not declare them as references at all (but just `const std::string` or `const QString`).

Reply:
Thanks, fixed.